jenkins/build.sh

```diff
@@ -15,22 +15,27 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
```

Comment:
`conf/script/v2/ha_setup.py`

it should be bundled with pyinstaller 
Should we have diferent pyinstaller bundle file for v2

Reply:
So this file I added only for reference, In mini provisioner implementation task we are going to change this file as per requirements.